traversal/procedure/ProcedureVertex.java

```diff
@@ -153,19 +176,10 @@ public boolean overlaps(Thing other, Traversal.Parameters params) {
 
         @Override
         public <ORDER extends Order> Forwardable<? extends ThingVertex, ORDER> iterator(
-                GraphManager graphMgr, Traversal.Parameters parameters, ORDER order
+                GraphManager graphMgr, Traversal.Parameters parameters, ORDER order, boolean forceValueSort
```

Comment:
we can force sorting by value by passing a boolean in, which will convert `Attribute` vertex into `Value` vertices and produce value-ordered vertices